workers/data_refinery_workers/downloaders/management/commands/queue_downloader.py

```diff
@@ -0,0 +1,47 @@
+from django.core.management.base import BaseCommand
```

Comment:
I put it inside the command class itself, do you think I should have put it at the top of this file instead?